admin_manual/index.rst

```diff
@@ -1,7 +1,3 @@
-============
-Introduction
-============
-
 Introduction
```

Comment:
@voroyam thanks for spotting and taking action. But wouldn't you remove this header instead?